integreat_cms/cms/templates/_base.html

```diff
@@ -101,7 +101,7 @@
          class="w-56 h-full fixed left-0 inset-y-0 text-gray-200 bg-gray-700 z-50 transform -translate-x-full lg:translate-x-0 motion-safe:transition-transform">
         <div class="w-full p-1 h-14 bg-gray-800">
             <a href="{% url 'public:region_selection' %}">
-                <div class="h-full w-full bg-{{ BRANDING }}-logo-white hover:bg-{{ BRANDING }}-logo-hover bg-contain bg-center bg-no-repeat">
+                <div class="h-full w-full bg-{{ BRANDING }}-logo-white hover:bg-{{ BRANDING }}-logo-hover ml-2 bg-contain bg-no-repeat">
```

Comment:
This change does not belong into this PR, does it? Could you rebase on develop again?